niceman/interface/tests/test_retrace_packages.py

```diff
@@ -33,7 +33,7 @@ def test_identify_packages():
     assert True
 
 
-@skip_if(not lexists(opj(dirname(__file__), pardir, pardir, pardir, '.git')))
+@skip_if(not lexists(opj(dirname(__file__)[:-25], sep, '.git')))
```

Comment:
what is the magical `[:-25]`?

Reply:
I changed the code to truncate the dir path rather than hop up the ".." pointers because, in my development environment, I use soft links to point the niceman executable to the various versions of conda python setups I have in ~/miniconda2/envs back to my active niceman git repo. The ".." hopping does not get back to the correct directory and fails the test_identify_myself() function

Reply:
and `25 == len("/niceman/interface/tests/")`, hm ok -- might worth a brief comment on that

Reply:
I'll put in your len("/niceman/interface/tests/") snippet instead of "25" so it makes more sense. And a comment, too